Flat/src/PandaAnalyzer.cc

```diff
@@ -1167,7 +1172,8 @@ void PandaAnalyzer::Run()
         
         TriggerEffs();
 
-        if (analysis->complicatedLeptons) 
+        if (analysis->complicatedLeptons ||
+	    analysis->complicatedPhotons)
```

Comment:
space

Reply:
I hope that's fixed because I didn't see anything strange in my editor. I now wrote the space one by one explicitly